index.js

```diff
@@ -59,7 +59,7 @@ module.exports = function(options) {
       // FIXME: origin can be promise
       origin = options.origin(ctx);
       if (!origin) {
-        return next();
```

Comment:
I think it is a breaking change here, this will make requests which not from browser all failed.